utils.py

```diff
@@ -1,9 +1,26 @@
 """Utility functions to be used in this Napp"""
 
+import requests
 from kytos.core import KytosEvent
 from napps.amlight.sdntrace_cp import settings
 
 
+def get_stored_flows(dpids: list = None, state: str = None):
+    """Get stored flows from flow_manager napps."""
+    api_url = f'{settings.FLOW_MANAGER_URL}/stored_flows'
+    if dpids:
+        str_dpids = ''
+        for dpid in dpids:
+            str_dpids += f'&dpid={dpid}'
+        api_url += '/?'+str_dpids[1:]
+    if state:
+        char = '&' if dpids else '/?'
+        api_url += char+f'state={state}'
+    result = requests.get(api_url)
```

Comment:
No need to handle this on this PR, but let's map an enhancement to add a timeout to this request, and also to use `tenacity` with quick retries (0.1s - 1s) interval and max 3 tries or something like that, if this could also land this release that'd be great.

Reply:
@viniarck I opened an issue for this.

Reply:
Awesome. Thanks, Gretel. Let me close this thread.